### ISSUE TITLE
[stacks-coordinator] DKG CLI command

### ISSUE THREAD

**Issue by netrome** [Trust-Machines/core-eng#17]
DoD: The coordinator supports in addition to running an event loop, one-off commands such as "process this particular peg-in request".

For alpha Needs to trigger DKG only.
Not for alpha: process peg out individual.

Time estimate: 1 day

Depends on #15

**Comment by stjepangolemac** [comment]
I want to clarify what needs to be done and what's already done.

1. The current coordinator code already has a CLI setup with the right commands and their logic, I suppose there's nothing to add there?
2. Do we just want to focus on the DKG one before alpha is ready to go?
3. Does this depend on https://github.com/Trust-Machines/core-eng/issues/61 ?

cc @netrome
